i18n/es.toml

```diff
@@ -65,25 +65,19 @@ other = "Volver a la vista normal de esta página"
 other = "Imprimir la sección entera"
 
 # Community
-[community]
-other = "community"
 [community_join]
-other = "Join the"
+other = "Únete a la comunidad {{ .Site.Title }}"
 [community_introduce]
-other = "is an open source project that anyone in the community can use, improve, and enjoy. We'd love you to join us! Here's a few ways to find out what's happening and get involved."
+other = "{{ .Site.Title }} es un proyecto de código abierto que cualquiera en la comunidad puede usar, mejorar y disfrutar. ¡Nos encantaría que te unas a nosotros! Aquí hay algunas maneras de averiguar lo que está sucediendo y participar."
 [community_learn]
-other = "Learn and Connect"
+other = "Aprende y Conéctate"
 [community_using]
-other = "Using or want to use"
-[community_find_out]
-other = "Find out more here"
+other = "¿Usas o quieres usar {{ .Site.Title }}, Encuentre más información aquí:"
```

Comment:
I think this is based on @eyllanesc  PR #866.

The first part of the sentence you refer to the user using the second person informal "¿Usas o quieres..." then switch to second person formal "Encuentre" instead of "Encuentra". Y would prefer to keep it informal throughout.

Reply:
Change to:

```toml
other = "¿Usas o quieres usar {{ .Site.Title }}? Encuentre más información aquí:"
```